admin/local/docker/compose/docker-compose.yml

```diff
@@ -98,6 +106,37 @@ services:
               source: volume_worker_0_mariadb_run
               target: /var/run/mysqld  # This is where the mariadb container puts the socket file
         network_mode: "service:worker-xrootd-0"
+    worker-wkr-0:
```

Comment:
Could be just `worker-0`

Reply:
I went with `worker-wkr` as I can search on that and `worker_wkr` and find just about everything associated with it. Searching on `worker` returns far too many unrelated things. I'm not set on the name, but something that can be found with grep will make life easier in the future.